src/entries/popup/hooks/send/useSendAsset.ts

```diff
@@ -65,7 +66,15 @@ export const useSendAsset = () => {
   );
 
   const allAssets = useMemo(
-    () => [...assets, ...customNetworkAssets],
+    () =>
+      sortedUniqBy(
+        [...assets, ...customNetworkAssets].sort(
+          (a: ParsedUserAsset, b: ParsedUserAsset) =>
+            parseFloat(b?.native?.balance?.amount) -
```

Comment:
`parseFloat(undefined)` is `NaN`, NaN is treated as 0 in `sort`, not sure if it matters just to be sure it's expected, I can think it may happen of a token where balance.amount is undefined somehow is sorted above another because of this

Reply:
Fair point but even with NaN it's the expected behavior (undefined treated as 0 and pushed to the end)